source/material_model/steinberger.cc

```diff
@@ -189,11 +194,14 @@ namespace aspect
       // For an explanation on this formula see the Steinberger & Calderwood 2006 paper
       const double vis_lateral_exp = -1.0*lateral_viscosity_lookup->lateral_viscosity(depth)*delta_temperature/(temperature*adiabatic_temperature);
       // Limit the lateral viscosity variation to a reasonable interval
+
       const double vis_lateral = std::max(std::min(std::exp(vis_lateral_exp),max_lateral_eta_variation),1/max_lateral_eta_variation);
 
       const double vis_radial = radial_viscosity_lookup->radial_viscosity(depth);
+      const double vis_compositional = MaterialUtilities::average_value (volume_fractions, prefactors, viscosity_averaging);
```

Comment:
```suggestion

      // Volume fraction averaged value of the composition-dependent viscosity prefactors
      const double vis_prefactor = MaterialUtilities::average_value (volume_fractions, prefactors, viscosity_averaging);
```

Reply:
I wanted to keep it as vis_compositional since I am already renaming "prefactors" as "viscosity_prefactors" as you have suggested. What do you think?